### PR TITLE
Windows support added for build system

### PR DESCRIPTION
**Pull request by rmariuzzo**
I'm just another user using SML in SublimeText under Windows, so I just added the support for Windows.

### CODE REVIEW

**Comment by seanjames777** [comment]
What is sml.bat? Is that installed by the MSI or did you write that? We'd like things to work by default as much as possible. That could mean including scripts for windows if necessary but I don't see one included.

**Comment by rmariuzzo** [comment]
It was installed by the MSI, I didn't wrote that. It appears in `C:\Program Files (x86)\SMLNJ\bin\sml.bat`

**Comment by seanjames777** [comment]
And Sublime can find it?

File: SML.sublime-build [hunk 1]
```diff
@@ -6,4 +6,8 @@
     "osx": {
         "path": "/opt/local/bin:/usr/bin:/opt/local/sbin:/bin:/usr/sbin:/usr/local/bin"
     }
```
Comment (rmariuzzo) [inline SML.sublime-build:8]: I forgot to put a comma `,`, sorry I edited this file online to be faster.

**Comment by rmariuzzo** [comment]
Yes, Sublime Text 2 can find without any tweaks. The MSI put the _sml.bat_ file into `C:\Program Files (x86)\SMLNJ\bin` and this last is added to the PATH variable.

**Comment by seanjames777** [comment]
Great

**Pull request merged**